sdk/src/main/java/com/podio/sdk/androidasynchttp/AndroidAsyncHttpRequest.java

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
Definitely is not a requirement from Citrix, we are not doing in the API.
Even in that case, the right way is to do it only once in the Readme, and not for every class.